DotNetHowToAutocomplete/DotNetHowToAutocomplete/Controllers/HomeController.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+
+using Microsoft.Azure.Search;
+using Microsoft.Azure.Search.Models;
+using System.Configuration;
+
+namespace AutocompleteTutorial.Controllers
+{
+    public class HomeController : Controller
+    {
+        private static SearchServiceClient _searchClient;
+        private static ISearchIndexClient _indexClient;
+        private static string IndexName = "nycjobs";
+
+        public static string errorMessage;
+
+        private void InitSearch()
```

Comment:
All the initialization in this method is for static fields, so it should go in a static constructor.